app/src/main/java/com/example/theperiodpurse/ui/setting/SettingScreenLayout.kt

```diff
@@ -0,0 +1,179 @@
+package com.example.theperiodpurse.ui.setting
+
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowRight
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.example.theperiodpurse.R
+
+
+@Composable
+fun SettingScreenLayout(
+    modifier: Modifier = Modifier,
+    onNotificationClicked: () -> Unit,
+    onBackUpClicked: () -> Unit,
+    onDeleteClicked: () -> Unit,
+){
+   Column(modifier = modifier
+       .fillMaxSize()
+       .verticalScroll(rememberScrollState())) {
+       Text(
+           text = stringResource(R.string.tracking_preferences),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
```

Comment:
if we are using `top = 50.dp` and `start = 10.dp` in every Text composable, would it be a good idea if we defined something like `val textModifier = modifier.padding(top = 50.dp, start = 10.dp)` and reuse it in the Text composables?